api/src/modules/projects/projects.service.ts

```diff
@@ -238,10 +242,25 @@ export class ProjectsService extends AppBaseService<
             },
           });
 
+        const { costOutput, breakEvenCostOutput } = costs;
+        if (breakEvenCostOutput) {
+          const { breakEvenCost, breakEvenCarbonPrice } = breakEvenCostOutput;
+          createProjectDto.priceType = PROJECT_PRICE_TYPE.OPEN_BREAK_EVEN_PRICE;
+          createProjectDto.initialCarbonPriceAssumption = breakEvenCarbonPrice;
+          const project = new ProjectBuilder(
+            createProjectDto,
+            scoreCardRating,
+            breakEvenCost,
+            projectSize.sizeHa,
+          );
+          await this.projectRepository.save(project.build());
+        }
+
+        createProjectDto.priceType = PROJECT_PRICE_TYPE.MARKET_PRICE;
         const project = new ProjectBuilder(
           createProjectDto,
           scoreCardRating,
-          costs,
+          costOutput,
           projectSize.sizeHa,
         ).build();
         return this.repository.save(project);
```

Comment:
Here we are returning only the 'regular' project, as this can clash how your implementation of the backoffice works, and I am not sure what to do